.travis.yml

```diff
@@ -12,6 +12,8 @@ jdk:
   - default-jdk
 
 script:
+- yarn eslint
+- yarn dtslint
```

Comment:
why the two lints? Does the `@typescript-eslint/eslint-plugin` not cover it? What does `dtslint` do?

Reply:
1. `dtslint` checks for [additional rules](https://github.com/microsoft/dtslint/tree/master/docs). AFAIK, ESLint doesn't have cover these rules and doesn't perform analysis on declaration files. 

`dtslint` is a static code analysis for TS declaration files.

2. I added linters into Travis stage because when we launch Bacon it executes [linting script](https://github.com/okta/okta-react-native/blob/master/scripts/lint.sh). I met the issue when Travis showed me success status, but Bacon failed because of linting violation.